code/game/turfs/open/space/space.dm

```diff
@@ -198,11 +198,6 @@
 			puller = pulling
 			pulling = next_pulling
 
-		//now we're on the new z_level, proceed the space drifting
-		stoplag()//Let a diagonal move finish, if necessary
-		arrived.newtonian_move(arrived.inertia_dir)
-		arrived.inertia_moving = TRUE
```

Comment:
You're telling me we don't even fucking need any part of it?

Reply:
this code was added years ago and newtonian_move() no longer directly moves like it did when it was added, it just adds to a processing queue for movement. since theyre moving anyways newtonian_move() gets called in atom/movable/Moved()